source/use_case_northern_lights.rst

```diff
@@ -0,0 +1,128 @@
+Find latest satellite image describing cloud cover for visibility of Northern Lights
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+..
+
+Use Case Goal
+=============
+
+.. 
+
+A user (tourist guide) wants to see Northern lights, but the current location is
+cloudy. Find information about cloud cover in the nearby areas to find places
+with less clouds. This can be done with infrared satellite imagerey and or
+analysed cloud products.
+
+Actors
+======
+
+.. 
+
+- Random user with limited experience
+- Random user (tourist guide) doing this every evening to plan an excursion
+- Data owner (satellite)(Obsklim, FOU-FD)
+- Data provider (satellite)(Obsklim, NBS)
+
+
+Trigger
+=======
+
+.. 
+
+- User experience cloudy conditions and wants to find nearby areas with less clouds.
+- User planning an excursion want to check the recent cloud cover information to plan the excursion.
+
+
+Pre-conditions
+==============
+
+.. 
+
+- Satellite product for visualisation clouds (gridded) at night must be available for WMS
+- Satellite product for download in like netcdf(gridded) for a more experienced user to visualize in their prefered tool
```

Comment:
I suggest to change the text slightly and highlight that the data should be accessible. Will push a new commit soon.

Reply:
have pushed already...

Reply:
see abecc5b

Reply:
OK, the commit looks good